Source/WebCore/accessibility/isolatedtree/AXIsolatedObject.cpp

```diff
@@ -505,6 +587,29 @@ void AXIsolatedObject::setProperty(AXPropertyName propertyName, AXPropertyValueV
         m_propertyMap.set(propertyName, value);
 }
 
+void AXIsolatedObject::addPropertyFlag(AXPropertyFlag flag)
+{
+    m_propertyFlags.add(flag);
+}
+
+void AXIsolatedObject::removePropertyFlag(AXPropertyFlag flag)
+{
+    m_propertyFlags.remove(flag);
+}
+
+void AXIsolatedObject::setPropertyFlag(AXPropertyFlag flag, bool set)
+{
+    if (set)
+        m_propertyFlags.add(flag);
+    else
+        m_propertyFlags.remove(flag);
+}
+
+bool AXIsolatedObject::hasPropertyFlag(AXPropertyFlag flag) const
+{
+    return m_propertyFlags.contains(flag);
+}
+
```

Comment:
Maybe all these one line methods should be inline in the header file below the class declaration. Maybe that's now done by the compiler automatically.

Reply:
Sounds good. Inlined.